events/code_review_events/workflow.py

```diff
@@ -207,6 +241,13 @@ def publish_results(self, payload):
                 extras["treeherder_url"],
             )
 
+            # Store the push revision and build, so we can use it after bugbug
+            # selects tests to add.
```

Comment:
Every data will be lost on dyno reboot... that may be an issue here

Reply:
Yeah, we will miss some (like we do for monitoring), but that's OK for now.